src/main.js

```diff
@@ -20,8 +20,12 @@ pkg.require({
     HackToolbox: '0',
 });
 
+const GLib = imports.gi.GLib;
```

Comment:
The new hotness is to use destructuring: `const {GLib} = imports.gi;`

Reply:
Fixed.